toolkit/pythonit_toolkit/api/client.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Any, Dict, Optional
+
+import httpx
+import jwt
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+
+INTERNAL_JWT_SECRET = os.getenv("INTERNAL_JWT_SECRET", "a-very-secret-secret")
+
+
+class Client:
+    url = "/internal-api"
+
+    async def execute(
+        self,
+        query: str,
+        variables: Optional[Dict[str, Any]] = None,
```

Comment:
so I am thinking here as input you should accept the service it is calling, the service is it calling from, variables

Reply:
Yeah I add them here in the execute method. So any time we want to call a query we have to include issuer and audience. I thought that maybe in the future can be the case where we want to use the same client to call different services...